freshrss/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: freshrss_server_1
+      APP_PORT: 80
+
+  server:
+    image: linuxserver/freshrss:1.21.0@sha256:de37e3f8f79c8c0fa00368784baa14d183385e34d168c1749dc69adcde6e45ec
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
```

Comment:
I checked and it's possible to change the timezone in the app which is great.

Reply:
I have removed the `TZ` variable as is not required.